projects/api_private/pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["hatchling", "hatch-polylith-bricks"]
+build-backend = "hatchling.build"
+
+[project]
+name = "api_private"
+version = "0.1.0"
+requires-python = ">=3.10"
+
+dependencies = [
+    # core
+    "fastapi>=0.115.12",
+    "uvicorn>=0.34.2",
+
+    # config / validation
+    "pydantic-settings>=2.9.1",
+    "python-dotenv>=1.0.1",
```

Comment:
pydantic-settings should import python-dotenv, and i don't think we use python-dotenv directly, do we?

Reply:
searched and didnt see any load env, removed it, still built